uint32 idmoving
uint8 novelty
uint16 reliability
float64 latitude
float64 longitude
float64 altitude
